cmd/agree/agree_test.go

```diff
@@ -26,9 +27,11 @@ var testTerms = "Test Terms"
 
 type agreeSuite struct {
 	client *mockClient
+	coretesting.FakeJujuXDGDataHomeSuite
```

Comment:
Anything wrong with FakeJujuHomeSuite?
The fact that it's in xdg data is surely just an implementation
detail?